test_oauth_signup.py

```
@@ -0,0 +1,150 @@
+    """Custom exception for OAuth test failures"""
+    pass
+
+async def verify_oauth_config():
+    """Verify OAuth configuration settings"""
+    print("\n🔍 Verifying OAuth Configuration...")
+    
+    base_url = "https://cryptouniverse.onrender.com"
+    oauth_config_url = f"{base_url}/api/v1/auth/oauth/config"
+    
+    try:
+        response = requests.get(oauth_config_url)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)